### PR TITLE
Nanoleaf

### PR DESCRIPTION
**Pull request by roderm**
### Features
- Control Power, Brightness and Color
- Select an Effect (Scene/Ambiente)
- Set fade for Brightness

### Setup
- After adding plugin press Power-Button on your Nanoleaf for 5-7 seconds until dots start flashing
- Start pairing on your bridge.

### CODE REVIEW

File: list.json [hunk 1]
```diff
@@ -248,6 +247,27 @@
...
+    "checksum": "ba07f21f378209dd633a382c48fd7dfc4e3ee37e94e6f0fb4c1be0ec8cefa4c4",
+    "packages": {
+      "any": {
+        "version": "0.1.0",
+        "url": "https://github.com/roderm/nanoleaf-adapter/releases/download/v0.1.0/nanoleaf-adapter-0.1.0.tgz",
+        "checksum": "ba07f21f378209dd633a382c48fd7dfc4e3ee37e94e6f0fb4c1be0ec8cefa4c4"
+      }
+    },
+    "api": {
+      "min": 1,
```
Comment (mrstegeman) [inline list.json]: After inspecting your add-on, your API minimum needs to be 2. You'll need to update your package.json and this file as well.

Comment (mrstegeman) [reply list.json]: After changing your min, you can remove the top-level `version`, `url`, and `checksum` from this entry, as they only need to be inside the `packages.any` object for API 2.

File: list.json [hunk 2]
```diff
@@ -248,6 +247,27 @@
       "max": 2
     }
   },
+  {
+    "name": "nanoleaf-adapter",
+    "display_name": "Nanoleaf",
+    "description": "Controll your Nanoleaf aurora",
```
Comment (mrstegeman) [inline list.json]: s/Controll/Control/

**Comment by mrstegeman** [comment]
Thank you!

**Comment by roderm** [comment]
Thank you for publishing my work.

**Pull request merged**